packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -454,17 +484,50 @@ export default {
         }
     }
 
+    &.c-megaModal-modeRTL {
+        .c-megaModal-closeBtn {
+            right: f.spacing(d);
+            left: auto;
+            transform: scaleX(-1);
+        }
+    }
+
+    &.c-megaModal--hasChevronIcon {
+        .c-megaModal-closeBtn {
+            left: f.spacing(d);
+        }
+    }
+
     .c-megaModal-closeBtn--fixed {
         position: fixed;
     }
 }
 
+.c-megaModal-modeRTL {
+    & .c-megaModal-title {
+        margin: 0 f.spacing(f) 0 0;
+    }
+}
+
 .c-megaModal-title {
     margin: 0 f.spacing(f) 0 0;
 
     .is-text-aligned-center & {
         margin-left: f.spacing(e);
         margin-right: f.spacing(e);
     }
+
+    .c-megaModal--hasChevronIcon & {
+        margin: 0 0 0 f.spacing(f);
+    }
+
+    .c-megaModal-modeRTL & {
+        margin: 0 f.spacing(f) 0 0;
```

Comment:
Is this effectively the same rule as L508?

Reply:
Fixed.